### DISCUSSION TITLE
XCT Sample App release to App Store and Google Play

### DISCUSSION THREAD

**Discussion by GUOLDEV** [xamarin/XamarinCommunityToolkit#1707] [General]
@jfversluis @pictos @bijington XCT Sample App release to App Store and Google Play, I am happy to do that using my private account, I just need the green light from you guys.

**Comment by wagenheimer** [comment]
+1 for this... I always need to see the updated demos, and I often can't compile the Samples.

**Comment by highwingers** [comment]
that would be great. I cant even compile the damn thing. I just want to check out the Samples.

**Comment by bijington** [comment]
I believe we discussed this during a standup call a few months ago and decided it wasn't something we had the bandwidth to support.
